packages/deployment/test/test_lambda.js

```diff
@@ -87,7 +87,42 @@ test.serial('zipLambda: works for lambda using message adapter', async (t) => {
 });
 
 test.serial(
-  `zipLambda: for lambda using message adapter, no new file is generated 
+  `zipLambda: given an invalid zip file generated from a previous run,
```

Comment:
If string interpolation is not being performed, use single quotes instead of backticks.

Reply:
For the case of long, single-line strings, the project convention is to use single quotes.  If eslint complains about the length of the line, add `// eslint-disable-line max-len` to the end of the line.